vector/src/main/java/im/vector/app/features/location/LocationSharingViewModel.kt

```diff
@@ -211,6 +212,10 @@ class LocationSharingViewModel @AssistedInject constructor(
         )
     }
 
+    private fun handleShowMapLoadingError() {
+        setState { copy(loadingMapHasFailed = true) }
+    }
+
```

Comment:
Do you confirm that if there is an error for any reason, the map will never be renderer with success later (so we should reset this field)?

Reply:
Yes, I confirm. I checked and the error is triggered when calling `MapboxMap.setStyle()` and it is only called once in `onViewCreated` in the fragments with maps. I moved the setup from `onResume()` in `LiveLocationMapViewFragment`. See [3f7f724](https://github.com/vector-im/element-android/pull/6712/commits/3f7f72413815e5f8137641727ad1dc73afa0949c) It seems, it was done in `onResume()` previously to fix missing markers after rotating the device but I could not reproduce this issue after update.